app/components/c-product-detail.vue

```diff
@@ -6,7 +6,7 @@
 
       <!-- left area -->
       <div :class="b('main', {area: 'top' })">
-
+        <e-info-label :class="b('info')" v-if="erp.priceType" :price-type="erp.priceType" :price-type-end-date="erp.priceTypeEndDate"/>
```

Comment:
- [ ] Don't add a bem class direct to the placed component, better use a wrapper div and set there the bem class for the positioning of the component